googleapiclient/errors.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, reason, resp=None, content=None):
     self.reason = reason
 
   def __repr__(self):
+    if getattr(self.resp, 'status', None) is not None:
```

Comment:
I don't really get it. This code will only print `self.resp.status` if `self.resp.status` is not None.